core/src/main/java/com/redhat/lightblue/client/request/LightblueHealthRequest.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.lightblue.client.request;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.redhat.lightblue.client.http.HttpMethod;
+
+public class LightblueHealthRequest extends LightblueRequest{
+
+    private static final long serialVersionUID = 2479314164852663290L;
+    
+    private String diagnosticsUri;
+
+    public LightblueHealthRequest() {
+        this("diagnostics");
+    }
+    
+    public LightblueHealthRequest(String diagnosticsUri) {
+        super(HttpMethod.GET);
+        this.diagnosticsUri = diagnosticsUri;    
```

Comment:
It doesn't make sense to parameterize the path (it's not a URI) for either health or diagnostics because they are a fixed part of lightblue's public API, just like /find or /insert, etc.